pkg/services/team/teamimpl/store_test.go

```diff
@@ -33,454 +34,439 @@ func TestIntegrationTeamCommandsAndQueries(t *testing.T) {
 	if testing.Short() {
 		t.Skip("skipping integration test")
 	}
-	t.Run("Testing Team commands and queries", func(t *testing.T) {
-		sqlStore, cfg := db.InitTestDBWithCfg(t)
-		teamSvc, err := ProvideService(sqlStore, cfg, tracing.InitializeTracerForTest())
-		require.NoError(t, err)
-		testUser := &user.SignedInUser{
-			OrgID: 1,
-			Permissions: map[int64]map[string][]string{
-				1: {
-					ac.ActionTeamsRead:         []string{ac.ScopeTeamsAll},
-					ac.ActionOrgUsersRead:      []string{ac.ScopeUsersAll},
-					serviceaccounts.ActionRead: []string{serviceaccounts.ScopeAll},
-				},
+	sqlStore, cfg := db.InitTestDBWithCfg(t)
```

Comment:
This entire test changed so much that it's difficult to compare old vs new version. It will take me a while.